README.md

```diff
@@ -1,12 +1,50 @@
-# Substrate API Sidecar
 
-REST API service intended to run next to Substrate, exposing a limited set of endpoints over HTTP
-with meaningful responses.
+<br /><br />
+
+<div align="center">
+  <h1 align="center">@substrate/api-sidecar</h1>
+  <h4 align="center"> REST service that makes it easy to interact with blockchain nodes built using Substrate's FRAME framework.</h4>
+
+  <p align="center">
+    <a href="https://www.npmjs.com/package/@substrate/api-sidecar">
+      <img alt="npm" src="https://img.shields.io/npm/v/@substrate/api-sidecar" />
+    </a>
+    <a href="https://github.com/paritytech/substrate-api-sidecar/actions">
+      <img alt="Github Actions" src="https://github.com/paritytech/substrate-api-sidecar/workflows/pr/badge.svg" />
+    </a>
+    <a href="https://github.com/paritytech/substrate-api-sidecar/blob/master/LICENSE">
+      <img alt="GPL-3.0-or-later" src="https://img.shields.io/npm/l/@substrate/api-sidecar" />
+    </a>
+  </p>
+</div>
+
+<br /><br />
+
+## Quick start for @substrate/api-sidecar CLI npm package
+
+```bash
+# Globally install the package
```

Comment:
Does it have to be installed in the global registry? If yes, let's mention that in the comment. If not, mention that too. :)